src/python_testing/TC_EEVSE_2_3.py

```diff
@@ -0,0 +1,445 @@
+
+    def pics_TC_EEVSE_2_3(self):
+        """ This function returns a list of PICS for this test case that must be True for the test to be run"""
+        return ["EEVSE.S", "EEVSE.S.F00"]
+
+    def steps_TC_EEVSE_2_3(self) -> list[TestStep]:
+        steps = [
+            TestStep("1", "Commissioning, already done",
+                     is_commissioning=True),
+            TestStep("2", "TH reads TestEventTriggersEnabled attribute from General Diagnostics Cluster.",
+                     "Verify value is 1 (True)"),
+            TestStep("3", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for Basic Functionality Test Event.",
+                     "Verify Command response is Success"),
+            TestStep("4", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for EVSE TimeOfUse Mode Test Event.",
+                     "Verify Command response is Success"),
+            TestStep("5", "TH sends TestEventTrigger command to General Diagnostics Cluster on Endpoint 0 with EnableKey field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER_KEY and EventTrigger field set to PIXIT.EEVSE.TEST_EVENT_TRIGGER for EV Plugged-in Test Event.",
+                     "Verify Command response is Success and event EEVSE.S.E00(EVConnected) sent"),
+            TestStep("6", "TH sends ClearTargets.",
+                     "Verify Command response is Success"),
+            TestStep("6a", "TH reads NextChargeStartTime attribute.",
+                     "Verify value is null."),
+            TestStep("6b", "TH reads NextChargeTargetTime attribute.",
+                     "Verify value is null."),
+            TestStep("6c", "TH reads NextChargeRequiredEnergy attribute.",
+                     "Verify value is null."),
+            TestStep("6d", "TH reads NextChargeTargetSoC attribute.",
+                     "Verify value is null."),
+            TestStep("7", "TH sends GetTargets.",
+                     "Response EEVSE.S.C00.Tx(GetTargetsResponse) sent with no targets defined."),
+            TestStep("8", "TH sends SetTargets with DayOfTheWeekforSequence=0x7F (i.e. having all days set) and a single ChargingTargets={TargetTime=1439, TargetSoC=null, AddedEnergy=25000000}.",
+                     "Verify Command response is Success"),
+            TestStep("8a", "TH reads NextChargeStartTime attribute.",
+                     "Verify value is null."),
+            TestStep("8b", "TH reads NextChargeTargetTime attribute.",
+                     "Verify value is null."),
+            TestStep("8c", "TH reads NextChargeRequiredEnergy attribute.",
+                     "Verify value is AddedEnergy above."),
```

Comment:
Currently the code below has a TODO which say expect NULL instead of 250000000

Reply:
Updated the test - we expected AddedEnergy to be NULL as EVSE is not enabled